packages/graphql-transformers-e2e-tests/src/__tests__/SubscriptionsWithAuthTest.e2e.test.ts

```diff
@@ -610,7 +627,9 @@ afterAll(async () => {
 // tests using cognito
 test('Test that only authorized members are allowed to view subscriptions', async () => {
   // subscribe to create students as user 2
-  const observer = GRAPHQL_CLIENT_2.subscribe({
+  reconfigureAmplifyAPI('AMAZON_COGNITO_USER_POOLS');
```

Comment:
Do we need to reconfigure the auth mode at every test? Since we can specify the auth mode when executing the operation.

Reply:
I do, it was erring out saying `subscription are only for Appsync`